BDDComicsList/App Modules/ComicsList/ComicsListRemoteData.swift

```diff
@@ -19,18 +18,25 @@ class ComicsListRemoteData {
     }
 }
 
+extension URL {
```

Comment:
Should this extension be `fileprivate`? If not, I would suggest to move it to a separate file

Reply:
It should. Thanks!